src/pamiq_core/console/cui.py

```diff
@@ -0,0 +1,141 @@
+                    groups[doc] = []
+                groups[doc].append(cmd_name)
+        # Define showing order
+        _doc_order = [
+            "Show all commands and details.",
+            "Pause the AMI system.",
+            "Resume the AMI system.",
+            "Save a checkpoint.",
+            "Shutdown the AMI system.",
+            "Exit the console.",
+        ]
+        assert set(_doc_order) == set(groups.keys())
+        # Show details in the format f"{c}/{command} {Docstring}"
+        for doc in _doc_order:
+            cmds = "/".join(sorted(groups[doc], key=len))
+            print(f"{cmds:<11} {doc}")
+
+    def do_h(self, arg: str) -> None:
+        """Show all commands and details."""
+        self.do_help(arg)
+
+    def do_pause(self, arg: str) -> None:
+        """Pause the AMI system."""
+        response = requests.post(f"http://{self._host}:{self._port}/api/pause")
+        print(json.loads(response.text)["result"])
+
+    def do_p(self, arg: str) -> None:
+        """Pause the AMI system."""
+        return self.do_pause(arg)
+
+    def do_resume(self, arg: str) -> None:
+        """Resume the AMI system."""
+        response = requests.post(f"http://{self._host}:{self._port}/api/resume")
+        print(json.loads(response.text)["result"])
+
+    def do_r(self, arg: str) -> None:
+        """Resume the AMI system."""
+        return self.do_resume(arg)
+
+    def do_shutdown(self, arg: str) -> bool:
+        """Shutdown the AMI system."""
+        confirm = input("Confirm AMI system shutdown? (y/n): ")
+        if confirm.lower() in ["y", "yes"]:
+            response = requests.post(f"http://{self._host}:{self._port}/api/shutdown")
+            print(json.loads(response.text)["result"])
+            return True
+        print("Shutdown cancelled.")
+        return False
+
+    def do_s(self, arg: str) -> bool:
+        """Shutdown the AMI system."""
+        return self.do_shutdown(arg)
+
+    def do_quit(self, arg: str) -> bool:
+        """Exit the console."""
+        return True
+
+    def do_q(self, arg: str) -> bool:
+        """Exit the console."""
+        return self.do_quit(arg)
+
+    def do_ckpt(self, arg: str) -> None:
+        """Save a checkpoint."""
+        response = requests.post(f"http://{self._host}:{self._port}/api/save-state")
+        print(json.loads(response.text)["result"])
+
+    def do_c(self, arg: str) -> None:
+        """Save a checkpoint."""
+        self.do_ckpt(arg)
+
+    @override
+    def postcmd(self, stop: bool, line: str) -> bool:
+        self._fetch_status()
+        return stop
+
+    def _fetch_status(self) -> str:
+        try:
+            response = requests.get(f"http://{self._host}:{self._port}/api/status")
+        except requests.exceptions.ConnectionError:
+            self.prompt = "ami (offline) > "
+            return "offline"
+        status = json.loads(response.text)["status"]
+        self.prompt = f"ami ({status}) > "
```

Comment:
単に`system`とする方が良いと思う。amiよりいとが伝わりやすい
```suggestion
            self.prompt = "system (offline) > "
            return "offline"
        status = json.loads(response.text)["status"]
        self.prompt = f"system ({status}) > "
```

Reply:
具体性をより重視して`PAMIQ-console`にしといた
https://github.com/MLShukai/pamiq-core/pull/210/commits/87780646f97b3684379e70dfc10276cc8a3e9c94

Reply:
う〜ん、それなら単に `pamiq` とするのが良いかもな。pamiq consoleがofflineなわけではないし

Reply:
pamiq consoleがWebAPIと繋がってないならpamiq consoleがofflineと呼んでしまっても良さそうな気はする　それにpamiq自体がofflineというわけではなさそうだし

Reply:
確かに。